8주차/14906/14906_java_연로그.java

```diff
@@ -0,0 +1,68 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class Main {
+
+    private static final StringBuilder sb = new StringBuilder();
+
+    public static void main(String[] args) throws IOException {
+        // input
+        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+        int n = Integer.parseInt(br.readLine());
+
+        sb.append("SLURPYS OUTPUT\n");
+        for (int i = 0; i < n; i++) {
+            String string = br.readLine();
+            if (isSlurpy(string)) {
+                sb.append("YES\n");
+            } else {
+                sb.append("NO\n");
+            }
+        }
+        sb.append("END OF OUTPUT\n");
+        System.out.println(sb);
+        br.close();
+    }
+
+    private static boolean isSlurpy(String string) {
+        int index = string.lastIndexOf('C');
+        if (index == -1) {
+            if (string.startsWith("AH")) {
+                return isSlump(string.substring(2));
+            }
+            return false;
```

Comment:
```suggestion
            return string(startsWith("AH") && isSlump(string.substring(2));
```

java에서 `lastIndexOf`, `startsWith` 문법 새롭게 알고 갑니다 👍